src/util/sync.js

```diff
@@ -49,11 +49,14 @@ export const sync = (thoughtIndexUpdates = {}, contextIndexUpdates = {}, { local
       }
       localForage.setItem('lastUpdated', lastUpdated)
     })
+
+    // recentlyEdited
+    localForage.setItem('recentlyEdited', recentlyEditedUpdates)
```

Comment:
As an argument, `recentlyEditedUpdates` should be optional, as there are many places in the code where `sync` is called just with `thoughtIndexUpdates`. The code as written will inadvertently clear the `recentlyEditedUpdates` list in these cases. Try entering an empty new thought and refreshing the page to reproduce.

I would change `recentlyEditedUpdates` to `recentlyEdited` in the signature like this:

`export const sync = (thoughtIndexUpdates = {}, contextIndexUpdates = {}, { local = true, remote = true, state = true, forceRender, updates, recentlyEdited, callback } = {}) => {`

And then only update it if a value was passed in.